iFixit/react-commerce
# Fix type errors in `@ifixit/ui`

- fix: move `IfixitImage` to `@ifixit/ui`
- fix: drop `packages/ui/footer/guide`

## QA

Do a quick browse of the site, and make sure images load as usual and there are no new console errors related to images.

@hcutrone

Connects #695 and #733
